client/extensions/woocommerce/state/sites/settings/general/selectors.js

```diff
@@ -111,3 +111,27 @@ export const areTaxCalculationsEnabled = ( state, siteId = getSelectedSiteId( st
 	}
 	return 'yes' === taxesEnabled.value;
 };
+
+export const isStoreNoticeEnabled = ( state, siteId = getSelectedSiteId( state ) ) => {
+	const generalSettings = getRawGeneralSettings( state, siteId );
+	const noticeEnabled = find( generalSettings, { id: 'woocommerce_demo_store' } );
+	if ( ! noticeEnabled ) {
+		return false;
+	}
+	if ( ! ( 'value' in noticeEnabled ) ) {
+		return false;
+	}
+	return 'yes' === noticeEnabled.value;
+};
+
+export const getStoreNotice = ( state, siteId = getSelectedSiteId( state ) ) => {
+	const generalSettings = getRawGeneralSettings( state, siteId );
+	const notice = find( generalSettings, { id: 'woocommerce_demo_store_notice' } );
+	if ( ! notice ) {
+		return null;
+	}
+	if ( ! ( 'value' in notice ) ) {
```

Comment:
```js
return notice.value || null;
```
Might be shorter? Just a small nit, up to you

Reply:
Sure - makes sense - removed there (and above) in https://github.com/Automattic/wp-calypso/pull/20721/commits/3647e0012fcad2b35f8d9d5c139f5e2d157737d1